README.md

```diff
@@ -55,6 +55,22 @@ SSL is enabled by default. You can disable it by passing a empty option list `[]
 TPusherServer.Create('app_id', 'app_key', 'secret', '', []);
 ```
 
+### Async
+
+To use async request you should use the `TAsyncPusherServer` class:
+```
+AsyncPusherServer := TAsyncPusherServer.Create('app_id', 'app_key', 'secret');
```

Comment:
I don't think this will be usefull without the `OnError` event handler, why not add the OnError as a parameter in the constructor?